openapi.yml

```diff
@@ -241,7 +232,7 @@ components:
         - hasAdminPolicy
     AdminPolicy:
       type: object
-      additionalProperties: false
+      additionalProperties: true
```

Comment:
Can we get away with not accepting extra properties? I think this is a valuable safeguard against submitting incorrect data.

Reply:
I agree. However, in some cases it causes committee problems. If I recall correctly, it is in endpoints that have parameters.